projects/plugins/jetpack/tests/e2e/bin/env.sh

```diff
@@ -25,6 +25,11 @@ reset_env() {
 configure_wp_env() {
 	if [ "$GUTENBERG" == "latest" ]; then
 		yarn wp-env run tests-cli wp plugin install gutenberg --activate
+	elif [ "$GUTENBERG" == "pre-release" ]; then
+		GB_ZIP="wp-content/gutenberg.zip"
+		yarn wp-env run tests-wordpress "wp-content/plugins/jetpack-dev/tests/e2e/bin/container-setup.sh gb-prerelease $GB_ZIP"
+		yarn wp-env run tests-cli "wp plugin install $GB_ZIP"
```

Comment:
`wp plugin install --activate` does not work as expected for some reason. That's why I am activating it in the line below.